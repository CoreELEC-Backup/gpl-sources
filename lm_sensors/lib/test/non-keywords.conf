# these are not keywords
ignored

labs

extra
stuff

