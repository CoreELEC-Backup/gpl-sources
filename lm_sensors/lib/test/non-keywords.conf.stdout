2: ERROR
3: EOL
4: ERROR
5: EOL
6: ERROR
7: EOL
7: ERROR
8: EOL
9: EOF
